services/src/libs/sentry.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * The sentry module is responsible for passing messages along to
+ * the sentry server.
+ */
+
+const logger    = require('winston');
+const Raven     = require('raven');
+
+
+class Sentry {
+  static initialize(sentryUrl) {
+    if (!sentryUrl) {
+      logger.error('No sentry url defined.');
+      return;
+    }
+    Raven.config(sentryUrl).install();
+  }
+
+  static sendOutput(data) {
+    if (!data) {
+      logger.error('Missing data.');
+      return;
+    }
+    if (data.log.exception) {
+      Raven.captureException(new Error(data.log.message), {
+        level: data.log.level.toLowerCase(),
```

Comment:
The object passed to both `captureMessage` and `captureException` looks identical to me, perhaps that can be refactored.

Reply:
it looks different in sentry which is why it's called differently.